packages/syft/src/syft/service/worker/utils.py

```diff
@@ -62,25 +70,26 @@ def run_container_using_docker(
 
 
 def run_containers(
+    client: docker.DockerClient,
     pool_name: str,
     worker_image: SyftWorkerImage,
     number: int,
     orchestration: WorkerOrchestrationType,
 ) -> List[ContainerSpawnStatus]:
-    image_tag = worker_image.image_tag
+    image_tag = worker_image.image_identifier
 
     results = []
     if not orchestration == WorkerOrchestrationType.DOCKER:
         return SyftError(message="Only Orchestration via Docker is supported.")
 
-    with contextlib.closing(docker.from_env()) as client:
```

Comment:
This seems inconsistent with `dev` branch. This line shouldn't be deleted, and the next loop should be within this scope.

Reply:
Thanks @shubham3121. It was added back